src/AltinnCore/Common/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -217,16 +218,25 @@ public async Task<Repository> GetRepository(string owner, string repository)
                 }
             }
 
+            var watch = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
Is this watch necessary ? Can't see that it is stopped at any point.

Reply:
No worries, just found the stop myself.. 
but does it handle the use of the same variable name okay ?

Reply:
No worries, just found the stop myself.. 
but does it handle the use of the same variable name okay ?

Reply:
It's actually stopped out of the if statement.  But i guess it will give wrong results. I will change the variable name